django_redis/client/default.py

```diff
@@ -568,8 +568,8 @@ def make_pattern(self, pattern, version=None, prefix=None):
     def close(self, **kwargs):
         if getattr(settings, "DJANGO_REDIS_CLOSE_CONNECTION", False):
             for i in range(len(self._clients)):
-                for c in self._clients[i].connection_pool._available_connections:
-                    c.disconnect()
+                client = self._clients[i]
+                self._disconnect_client(client)
```

Comment:
I do not think a separate method is needed, just check if it is `None` and call `.disconnect()` from the connection_pool

Reply:
OK I do

Reply:
As for cache.delete("other_key"), it is needed to pass these tests. I solve this problem in this PR https://github.com/jazzband/django-redis/pull/494